package.json

```diff
@@ -1,15 +1,13 @@
 {
   "name": "bgr-ember-pagination",
   "version": "0.5.1",
-  "description": "Pagination component with out of the box support for JSON API.",
+  "description": "Pagination component",
```

Comment:
I'll add a better description if you write me one 😄

Reply:
Will think of something!